sql/analyzer/indexed_joins.go

```diff
@@ -580,6 +580,18 @@ func convertAntiToLeftJoin(m *memo.Memo) error {
 				sch = pkt.PrimaryKeySchema().Schema
 			}
 
+			// column matching logic is dependent on the schema not being a sql.ProjectedTable
+			if ta, ok := srcNode.(*plan.TableAlias); ok {
+				if rt, ok := ta.Child.(*plan.ResolvedTable); ok {
+					if t := rt.UnderlyingTable(); t != nil {
+						if pt, ok := t.(sql.ProjectedTable); ok {
+							tmpTbl := pt.WithProjections(nil)
```

Comment:
`sql.PrimaryKeyTable` is how we do this in other places, i'm pretty sure the memo RelProps has this already